packages/core/test/unit/query-interface/create-table.test.ts

```diff
@@ -70,15 +70,8 @@ describe('QueryInterface#createTable', () => {
   }
 
   it('supports sql.uuidV1 default values', async () => {
-    const localSequelize =
-      dialect.name === 'mysql'
-        ? createSequelizeInstance({
```

Comment:
Do we want to add an internal unit test for this?

Reply:
The minimum version of MySQL supported is 8.0.19 which is why I removed this.